rasa/cli/interactive.py

```diff
@@ -55,7 +55,6 @@ def interactive(args: argparse.Namespace):
     _set_not_required_args(args)
 
     if args.model is None:
-        check_training_data(args)
         zipped_model = train.train(args)
```

Comment:
we should still exit in case this returns `None`

Reply:
Good point!